recipe/patches/0001-Rename-qpOASES-namespace-to-casadi_qpOASES.patch

```diff
@@ -1,18 +1,28 @@
+From d2e2589f04adaa14cf4908101ae6ac84ad12b02f Mon Sep 17 00:00:00 2001
+From: Joris Vaillant <joris.vaillant@inria.fr>
+Date: Sun, 03 Jun 2024 15:58:41 +0200
+Subject: [PATCH 01/05] Rename qpOASES namespace to casadi_qpOASES
```

Comment:
I do not think this is a big problem if this metadata is not correct, but can you explain how this was generated? I am not sure, but I do not think this patch was prepare by Joris Vaillant.

Reply:
Okay, I see that you added the patch in #77 back in 2022. Updated in 8cdd3f1c2a252b37048746dd3bb249fa067de6da, thanks for raising this point.